SECURITY.md

```diff
@@ -0,0 +1,52 @@
+Our top priority is keeping our customers' data safe. If you have found an issue in our systems, please reach out to us.
+
+## Reporting a Vulnerability
+
+If you believe you have found a vulnerability, please disclose by contacting us: security@highlight.io
+
+Please try your best to describe a clear and realistic impact for your report.
+
+## Supported versions
+
+| Version     | Supported |
+|-------------|-----------|
+| main branch | ️✅        |
+| any other   | ❌         |
+
+
+## Vulnerabilities we care about 🫣
+> Note: Please use a [self-hosted instance](https://www.highlight.io/docs/getting-started/self-host/self-hosted-hobby-guide) to perform any tests. Do not use the production app.highlight.io product for security testing.
+- Remote command execution
+- SQL Injection
+- Authentication bypass
+- Privilege Escalation
+- Cross-site scripting (XSS)
+- Performing limited admin actions without authorization
+- CSRF
+
+You can learn more about our triaging process [here](https://www.highlight.io/docs/general/company/open-source/contributing).
+
+## Non-Qualifying Vulnerabilities
+
+We consider the following out of scope, though there may be exceptions.
+
+- DMARC configuration
```

Comment:
where did we get all of these items? was it inspired by anything else?

Reply:
these specifically were inspired by https://github.com/chatwoot/chatwoot/blob/develop/SECURITY.md
DMARC config in particular was one i read about being a typical gotcha
happy to simplify this too though - we can just omit listing what vulnerabilities we care / don't care about.

Reply:
was just curious. this sounds good to me.